keps/sig-windows/1981-windows-privileged-container-support/README.md

```diff
@@ -628,24 +728,26 @@ in back-to-back releases.
 -->
 
 Alpha
-- Version of containerd: 	v1.5
-- Version of Kubernetes: 	Target 1.20 or 1.21  
-- Version of OS support: 	1809/Windows 2019 LTSC and 2004
-- Alpha Feature Gate for passing privilege flag to CRI
+
+- Version of containerd: v1.5
+- Version of Kubernetes: Target  1.21
+- Version of OS support: 1809/Windows 2019 LTSC and 2004
```

Comment:
this would 2019+ (including 20H2)?

Reply:
Should be, there shouldn't be any OS restriction here